uber/templates/registration/new_base.html

```diff
@@ -185,7 +260,6 @@
             <form method="post" id="new_checkin_{{ attendee.id }}" action="new_checkin">
```

Comment:
This is an existing bug, unrelated to this pull request, but this construct is not valid HTML. `<form>` is not a valid child of `<tr>` and `<td>` is not a valid child of `<form>`. The only children `<tr>` is allowed to have are `<td>` and `<th>`, and the only parent `<td>` is allowed to have is `<tr>`.

We would be better off merging the two columns, and making the `<form>` a child of the `<td>`:
```html
<td>
    <form method="post" id="new_checkin_{{ attendee.id }}" action="new_checkin">
      {% if c.NUMBERED_BADGES %}
      <input type="number" name="badge_num" size="5" style="width:5em" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %} />
      {% endif %}
      {{ csrf_token() }}
      <input type="hidden" name="id" value="{{ attendee.id }}" />
      <button class="btn btn-primary" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %}>Check In</button>
    </form>
</td>
```

As it stands this ends up being rendered by the browser like this:
```html
<form method="post" id="new_checkin_077fcb78-8c87-4b17-a180-18b960717c21" action="new_checkin"></form>
<td>
    <input type="number" name="badge_num" size="5" style="width:5em" disabled="">
</td>
<td>
    <input type="hidden" name="csrf_token" value="f143ff6b1246436c8e3e04e747e2d726">
    <input type="hidden" name="id" value="077fcb78-8c87-4b17-a180-18b960717c21">
    <button class="btn btn-primary" disabled="">Check In</button>
</td>
```

Which only happens to work because browsers are so tolerant of illegal HTML.